documents/src/pages/build-element/tutorial.md

```diff
@@ -98,15 +98,84 @@ For more information on composing templates, see [Lit](https://lit.dev/docs/comp
 
 ### Reactive Properties
 
-@> Coming soon!
+Reactive properties are properties that, when changed, can trigger an update cycle and re-calling rendering the component. Use `@property` decorator with type and options to define the property.
+
+```ts
+export class TodoElement extends BasicElement {
+  @property({ type: String })
+  name: String;
+}
+```
 
-### Styles & Theming
+Usually reactive property define as public by default. To define internal property that can trigger reactive update cycle use `@state` decorator. In Typescript, these should assign `private` or `protected` to define the type is scoped in the component.
 
-@> Coming soon!
+```ts
+export class TodoElement extends BasicElement {
+  @state({ type: Array })
+  protected data: String[] = [];
+}
+```
+
+For more information on reactive properties, see [Lit](https://lit.dev/docs/components/properties/).
+
+### Attributes
+
+While properties usage works on Javascript, attributes configure the element in HTML. Whether property or attribute trigger update cycle. By default, attribute is observed to reactive property. The observed attribute name informs on lowercase.
```

Comment:
```suggestion
While properties usage works on Javascript, attributes configure the element in HTML. Whether the property or the attribute has changed, it will trigger update cycle. By default, the attribute observed to reactive property and stated in lowercase.
```